pkg/utils/constants.go

```diff
@@ -90,3 +90,7 @@ const (
 	// Default port used to access vCenter.
 	DefaultVCenterPort string = "443"
 )
+
+const (
+	DataManagerBin string = "/data-manager-for-plugin:"
```

Comment:
`DataManagerRepository string = "data-manager-for-plugin"` makes much more sense

Reply:
Changed to "data-manager-for-plugin".